include/tdi/common/c_frontend/tdi_attributes.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright(c) 2021 Intel Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this software except as stipulated in the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/** @file tdi_attributes.h
+ *
+ *  @brief Contains TDI Attribute APIs
+ */
+#ifndef _TDI_ATTRIBUTES_H
+#define _TDI_ATTRIBUTES_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+tdi_status_t tdi_attributes_set_value(tdi_attributes_hdl *attributes,
```

Comment:
const before the tdi_attributes_hdl

Reply:
I don't think we should be making the hdl in a setValue function as const